### ISSUE TITLE
RC9 Settings Page Issue

### ISSUE THREAD

**Issue by edolyne** [ethercreative/splash#5]
When navigating to the settings page in `RC9` the following error is encountered:

```
Internal Server Error
Argument 1 passed to craft\services\Fields::getLayoutById() must be of the type integer, null given, called in /***/***/***/***/***/vendor/ether/splash/src/Splash.php on line 79
```

**Comment by janhenckens** [comment]
I'm running into the same issue... 
Is this plugin abandoned?

**Comment by klick** [comment]
Same here.

**Comment by janhenckens** [comment]
@klick Allow me to point you to my version of this: https://github.com/studioespresso/craft3-unsplash

**Comment by klick** [comment]
Thank you, I already have your version installed which works very well. However, I’m missing the 

* Store the images author and a link to their Unsplash profile

feature in your version.

**Comment by alexjcollins** [comment]
Hey everyone, we'll take a look at the issue and let you know

**Comment by Tam** [comment]
Fixed in 3.0.3
